Source: from ArchLinux: https://projects.archlinux.org/svntogit/community.git/plain/trunk/set_wheel.patch?h=packages/cinnamon
Upstream: yes, GitHub issue #3576: https://github.com/linuxmint/Cinnamon/issues/3576
Reason: Use wheel group instead of relying on sudo

--- a/files/usr/share/cinnamon/cinnamon-settings-users/cinnamon-settings-users.py
+++ b/files/usr/share/cinnamon/cinnamon-settings-users/cinnamon-settings-users.py
@@ -254,13 +254,7 @@
         newpass = self.new_password.get_text()
         self.user.set_password(newpass, "")
         mask = self.group_mask.get_text()
-        if "nopasswdlogin" in mask:
-            subprocess.call(["gpasswd", "-d", self.user.get_user_name(), "nopasswdlogin"])
-            mask = mask.split(", ")
-            mask.remove("nopasswdlogin")
-            mask = ", ".join(mask)
-            self.group_mask.set_text(mask)
-            self.password_mask.set_text('\u2022\u2022\u2022\u2022\u2022\u2022')
+        self.password_mask.set_text('\u2022\u2022\u2022\u2022\u2022\u2022')
         self.destroy()
 
     def set_passwords_visibility(self):
@@ -876,11 +870,11 @@
             pixbuf = GdkPixbuf.Pixbuf.new_from_file_at_size("/usr/share/cinnamon/faces/user-generic.png", 48, 48)
             description = "<b>%s</b>\n%s" % (fullname, username)
             piter = self.users.append(None, [new_user, pixbuf, description])
-            # Add the user to his/her own group and sudo if Administrator was selected
+            # Add the user to his/her own group and wheel if Administrator was selected
             if dialog.account_type_combo.get_active() == 1:
-                subprocess.call(["usermod", username, "-G", "%s,sudo,nopasswdlogin" % username])
+                subprocess.call(["usermod", username, "-G", "%s,wheel" % username])
             else:
-                subprocess.call(["usermod", username, "-G", "%s,nopasswdlogin" % username])
+                subprocess.call(["usermod", username, "-G", "%s" % username])
             self.load_groups()
         dialog.destroy()
  
